modulemd/v2/tests/ModulemdTests/translation_helpers.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+# This file is part of libmodulemd
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Fedora-License-Identifier: MIT
+# SPDX-2.0-License-Identifier: MIT
+# SPDX-3.0-License-Identifier: MIT
+#
+# This program is free software.
+# For more information on the license, see COPYING.
+# For more information on free software, see
+# <https://www.gnu.org/philosophy/free-sw.en.html>.
+
+import os
+import sys
+
+try:
+    import unittest
+    import gi
+    gi.require_version('Modulemd', '2.0')
+    from gi.repository import Modulemd
+except ImportError:
+    # Return error 77 to skip this test on platforms without the necessary
+    # python modules
+    sys.exit(77)
+
+from base import TestBase
+
+
+class TestTranslationHelpers(TestBase):
+
+    def test_translation_catalog_from_index(self):
+        # Create a Modulemd.ModuleIndex object containing the YAML stream
+        index = Modulemd.ModuleIndex.new()
+        yaml_path = "../test_data/f29.yaml"
+        ret, failures = index.update_from_file(yaml_path, True)
+        self.assertIsNotNone(index)
+
+        catalog = get_translation_catalog_from_index(index)
+        self.assertTrue(catalog)
+        self.assertEquals(len(catalog), 34)
```

Comment:
Explain *why* the number 34 is correct in a comment, please. "Magic" numbers like this always need to be explained, or else if this test starts failing, the next person won't know what to do about it.